pkg/operator/webhookconfig.go

```diff
@@ -78,57 +78,59 @@ func (w *WebhookConfigUpdater) Start(stopCh <-chan struct{}) error {
 }
 
 // ValidatingWebhooks returns the validating webhook configurations.
-func (w *WebhookConfigUpdater) ValidatingWebhooks() ([]admissionregistrationv1beta1.ValidatingWebhook, error) {
-	failurePolicy := admissionregistrationv1beta1.Ignore
-	sideEffects := admissionregistrationv1beta1.SideEffectClassNone
+func (w *WebhookConfigUpdater) ValidatingWebhooks() ([]admissionregistrationv1.ValidatingWebhook, error) {
+	failurePolicy := admissionregistrationv1.Ignore
+	sideEffects := admissionregistrationv1.SideEffectClassNone
 
-	webhooks := []admissionregistrationv1beta1.ValidatingWebhook{
+	webhooks := []admissionregistrationv1.ValidatingWebhook{
 		{
-			Name: "clusterautoscalers.autoscaling.openshift.io",
-			ClientConfig: admissionregistrationv1beta1.WebhookClientConfig{
-				Service: &admissionregistrationv1beta1.ServiceReference{
+			Name:                    "clusterautoscalers.autoscaling.openshift.io",
+			AdmissionReviewVersions: []string{"v1", "v1beta1"},
```

Comment:
IIUC we want to just support v1 going forward as v1beta1 will be removed in 1.22
```suggestion
			AdmissionReviewVersions: []string{"v1"},
```

Reply:
From field description: `API server will try to use the first version in the list which it supports. `. I left v1beta as a safety check, v1 will be used anyway.

Reply:
I'm just wondering if there is any validation that checks these versions against a list of supported versions, eg it works in 1.21 because v1 and v1beta1 are both supported, but breaks in 1.22 because it no longer knows what v1beta1 is 🤔 

Given every cluster that we deploy this to will be 1.21 or higher (as we aren't backporting this) we can guarantee that the clusters will support `v1`,  I don't think it's necessary to keep the fallback